demisto_sdk/commands/lint/resources/installation_scripts/python_image.sh

```diff
@@ -9,17 +9,19 @@ cd /devwork
 chown -R :4000 /devwork/
 chmod -R 775 /devwork
 . /etc/os-release
+
 if [ "$ID" = "alpine" ]
 then
-    apk add --no-cache --virtual .build-deps python3-dev gcc build-base;
+    apk update && apk add --no-cache --virtual .build-deps python3-dev gcc build-base;
```

Comment:
why update?

Reply:
In old images the `alpine` is outdated and the `apk` (alpine package manager) needs to be updated